addon/components/dropdown-action.ts

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+import { scheduleOnce } from '@ember/runloop';
+import { tryInvoke } from '@ember/utils';
+export default class DropdownActionComponent extends Component {
+    bubbles: boolean = true;
+
+    click() {
+        //@ts-ignore
+        const ddActions = this.dd.actions;
+        tryInvoke(this, 'action', [event]);
+
+        if(ddActions) {
+            scheduleOnce('afterRender', ddActions, 'close');
+        }
+
+        return this.bubbles;
+    }
```

Comment:
This `click()` method needs to be cleaned up, and would probably rename it to `onClick`, and should become an `@action`. We can get rid of the `bubbles` stuff entirely (as thats a classic ember thing), also get rid of `tryInvoke` and just do that the TS-way:
```ts
this.args.action?.(event);
```

the method should also have a doc block comment documentation, and have its param typed, e.g. 
```ts
@action
onClick(event: Event) { ///...
```